### DISCUSSION TITLE
I Just Want to Know How Can I Out Two Files From Cirrus.

### DISCUSSION THREAD

**Discussion by ImSurajx** [ROM-builders/temporary#22914] [Q&A]
Like I Want To Out Pe Rom & Pe Recovery To .

**Comment by Apon77** [comment]
mm, for now there is no way, where the file exists? i mean which file to upload? i write script to upload multiple file by variable...

**Reply by hipexscape** [reply]
he need to upload this 
out/target/product/*/recovery.img

**Reply by ImSurajx** [reply]
@Apon77 out/target/product/lavender/recovery.img

**Reply by Apon77** [reply]
you can just download the rom in google cloud shell, then extract the recovery. its a bypass way for now. most of the time people do not need recovery.

**Comment by ImSurajx** [comment]
Okay
